subprojects/execution/src/main/java/org/gradle/internal/execution/history/changes/OutputFileChanges.java

```diff
@@ -22,15 +22,7 @@
 
 public class OutputFileChanges extends AbstractFingerprintChanges {
```

Comment:
I think we either should rename this to `DefaultOutputFileChanges`, or better rename `DefaultInputFileChanges` to `InputFileChanges`.

Reply:
We can't rename `DefaultInputFileChanges` to `InputFileChanges` since an interface with that name already exists. We don't have an interface for `OutputFileChanges`. I would leave this as is for now.